Sprint-3/2-practice-tdd/get-ordinal-number.test.js

```diff
@@ -11,3 +11,15 @@ const getOrdinalNumber = require("./get-ordinal-number");
 test("should return '1st' for 1", () => {
   expect(getOrdinalNumber(1)).toEqual("1st");
 });
+
+test("should return '11th' for 11", () => {
```

Comment:
Same as a previous comment regarding very slim test suite. Try to think of more test cases.